### ISSUE TITLE
Suggestion and Issue

### ISSUE THREAD

**Issue by Preminenc3** [TheIllusiveC4/ConstructsArmory#14]
All paper armor does not show up on the player's body or in the inventory equip slot, and end armors are duplicated in JEI.

Would it be possible to make armor for dragons and hippogriffs from Ice and Fire Mod?

**Comment by kierajreed** [comment]
Cool, but I think mod compatibility will only be considered after this mod is stable, IIRC.

**Comment by TheIllusiveC4** [comment]
1) This is the first time I've heard of that paper armor issue. Does it not show up at all or just not the right texture? Because there's just a placeholder texture for now. If it doesn't show up at all, I cannot replicate this issue. The paper armor shows up fine for my test install. Can you replicate it with just Construct's Armory, Tinkers' Construct, and Mantle installed? If you can, could you screenshot it for me so I can see for myself?

2) I am aware of the End armor being duplicated in JEI, but I have no idea what's causing the issue because it shows up correctly (only once) in the vanilla Creative tab. It has something to do with how JEI in particular fetches items. It's something I'm looking into, but with low priority since I don't think it's causing any particular issues.

3) Armor for dragons and hippogriffs from Ice and Fire would have to come way further down the line, and even then I think that would be best suited for someone else to handle as an add-on because I am not the best at designing armor.

**Comment by TheIllusiveC4** [comment]
I was too curious about the End armor and JEI stuff, so I ended up doing a deep dive on the matter and figured out that I just needed to implement some of the JEI API stuff. So this will be fixed in the next update.

Still will need more info on that paper armor bug though.

**Comment by Preminenc3** [comment]
I will get to you later today

**Comment by TheIllusiveC4** [comment]
Still waiting on an update for this. If no new information is given within the next week, I'll be forced to close this issue.

**Comment by Preminenc3** [comment]
Sorry will do it today I have been really busy

**Comment by Preminenc3** [comment]
It works actually, it was a problem with foamfix

**Comment by TheIllusiveC4** [comment]
Oh okay. That's a bit weird, but I guess it's not my problem anymore. Thanks for letting me know.
